docs/installation/instructions.rst

```diff
@@ -18,16 +26,25 @@ PyPI Installation
    $ python3 -m venv pulpvenv
    $ source pulpvenv/bin/activate
 
+.. note::
+
+   On some operating systems you may need to install a package which provides the ``venv`` package.
+   For example, on Ubuntu or Debian you need to run::
+
+   $ sudo apt-get install python3-venv
+
 3. Install Pulp::
 
    $ pip3 install pulpcore
 
-
 .. note::
 
-   To install from source, replace the pip3 install commands to specify a source install such as::
+   To install from source, clone the git repository, checkout the 3.0-dev branch, navigate to the
+   directory containing the ``setup.py`` file, and do a local, editable pip installation::
```

Comment:
It might be good to call out the 3 packages needed: pulpcore, plugin, and common

Reply:
I thought that too but since ```pulpcore``` depends on ```pulpcore-common``` and any plugin should (and currently does) depend on ```pulpcore-plugin```, I think it's fine to leave it out.

Reply:
+1